DICOM/src/Microsoft.Health.Dicom.Anonymizer.Core/DicomUtility.cs

```diff
@@ -0,0 +1,167 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Dicom;
+using EnsureThat;
+using Microsoft.Health.Anonymizer.Common.Models;
+
+namespace Microsoft.Health.Dicom.Anonymizer.Core
+{
+    /// <summary>
+    /// Utility functions are used to parse DICOM data including DA, DT, AS. The format for these VR is given in DICOM standard.
+    /// http://dicom.nema.org/medical/Dicom/2017e/output/chtml/part05/sect_6.2.html
+    /// </summary>
+    public static class DicomUtility
+    {
+        private static readonly Dictionary<string, AgeType> AgeTypeMapping = new Dictionary<string, AgeType>
+            {
+                { "Y", AgeType.Year },
+                { "M", AgeType.Month },
+                { "W", AgeType.Week },
+                { "D", AgeType.Day },
+            };
+
+        private const int AgeStringLength = 3;
+        private const int MaxAgeValue = 999;
```

Comment:
❔ [Question]

I remembered that the age threshold is 89, but the age input is still 3 digits, right?

Reply:
> age input is still 3 digits, right?

Yes. the format is like nnnY, nnnM, ... 😎